returnn/tf/layers/signal_processing.py

```diff
@@ -490,12 +570,13 @@ def get_out_data_from_opts(cls, fft_sizes, use_rfft=True, nr_of_channels=1, **kw
 
 class MultiChannelStftLayer(MultiChannelMultiResolutionStftLayer):
   """
-  The layer applys a STFT to every channel separately and concatenates the frequency domain vectors for every frame
+  The layer applys a STFT to every channel separately and concatenates the frequency domain vectors for every frame.
   """
   recurrent = True
   layer_class = "multichannel_stft_layer"
 
-  def __init__(self, frame_shift, frame_size, fft_size, window="hanning", use_rfft=True, nr_of_channels=1, pad_last_frame=False, **kwargs):
+  def __init__(self, frame_shift, frame_size, fft_size, window="hanning", use_rfft=True, nr_of_channels=1,
+               pad_last_frame=False, **kwargs):
```

Comment:
Note: For the function arguments in a function definition, I think the indentation is ok. But only for that.